app/views/_includes/summary-cards/qualifications.html

```diff
@@ -1,7 +1,28 @@
 {# Show this if the user says they have qualification to enter #}
 {% if data.anyQualifications == "Yes" %}
```

Comment:
move this if statement to above` {{ xGovukSummaryCard({ ` which is currently on line 25.

Reply:
Then delete lines 161-181